### ISSUE TITLE
a guide helping contributors to develop the library

### ISSUE THREAD

**Issue by piotr-s-brainhub** [brainhubeu/react-carousel#550]
Some contributors, who want to fix a bug or implement a new feature in react-carousel, don't know how to start so we should create a guide describing the meaning of particular directories/ files/ classes/ modules/ functions/ methods.

**Comment by Ckbhatia** [comment]
It is interesting ♥️

**Comment by piotr-s-brainhub** [comment]
@Ckbhatia 

Maybe you see something particular, you'd like to be explained in such a guide?

**Comment by Ckbhatia** [comment]
Yes, I would like to explain, how to get started with react carousel to contribute.

**Comment by piotr-s-brainhub** [comment]
@Ckbhatia 

So you don't understand the react-carousel code?

**Comment by Ckbhatia** [comment]
Yes, I didn't understand the folder structure for this.

You have a guide to getting started with documents. but no guide for react carousel itself.

**Comment by piotr-s-brainhub** [comment]
thanks

**Comment by briankosw** [comment]
I can attempt to work on this. I just recently did a pull request for a new feature #577, which meant that I had to do a good amount of digging around in the codebase. I am quite new to React (just started yesterday!) so I might need some help from more experienced users.

**Comment by piotr-s-brainhub** [comment]
@briankosw 

You're more than welcome to open a PR.
